### PR TITLE
{bio}[foss/2020b] EIGENSOFT v7.2.1

### PR DESCRIPTION
**Pull request by DirectorSloan**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by DirectorSloan** [comment]
Can this go into develop? Any help or tuning needed?

File: easybuild/easyconfigs/e/EIGENSOFT/EIGENSOFT-7.2.1-foss-2020b.eb [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+files_to_copy = ['bin', 'CONVERTF', 'EIGENSTRAT', 'POPGEN', 'README']
+
+fix_perl_shebang_for = ['bin/*.perl']
+
+sanity_check_paths = {
+    'files': ['bin/%s' % x for x in ["baseprog", "convertf", "eigenstrat", "eigenstratQTL"]],
+    'dirs': []
+}
+
+moduleclass = 'bio'
```
Comment (boegel) [inline easybuild/easyconfigs/e/EIGENSOFT/EIGENSOFT-7.2.1-foss-2020b.eb:54]:
```suggestion
# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
# Author: Pablo Escobar Lopez
# sciCORE - University of Basel
# SIB Swiss Institute of Bioinformatics
# 6.1.4 modified by:
# Adam Huffman
# The Francis Crick Institute
# 7.2.1 modified by:
# Tom Strempel
# Helmholtz-Centre for Environmental Research - UFZ

easyblock = 'MakeCp'

name = 'EIGENSOFT'
version = '7.2.1'

homepage = 'https://www.hsph.harvard.edu/alkes-price/software/'
description = """The EIGENSOFT package combines functionality from our population genetics methods (Patterson et al.
2006)  and our EIGENSTRAT stratification correction method (Price et al. 2006). The EIGENSTRAT method uses principal
components  analysis to explicitly model ancestry differences between cases and controls along continuous axes of
variation;  the resulting correction is specific to a candidate marker’s variation in frequency across ancestral
populations,  minimizing spurious associations while maximizing power to detect true associations. The EIGENSOFT
package has a built-in plotting script and supports multiple file formats and quantitative phenotypes."""

toolchain = {'name': 'foss', 'version': '2020b'}

source_urls = ['https://github.com/DReichLab/EIG/archive']
sources = ['v%(version)s.tar.gz']
patches = [
    '%(name)s-%(version)s_Fix_makefile_openblas.patch',
    ('%(name)s-%(version)s_restore-ploteig.patch', 0),
]
checksums = [
    'f09a46ec4b83c5062ec71eaca48a78f2373f1666fe23cbf17757150a679c8650',  # v7.2.1.tar.gz
    'e49e3754f2326210114fe5a731a77c7ffd240c8a9134eb8e8e1517bfe06c71e1',  # EIGENSOFT-7.2.1_Fix_makefile_openblas.patch
    '8a7a0273ae4d0d3ec0c9927facd41a1a43b8540725af3bd06e007cd86afaf9e0',  # EIGENSOFT-7.2.1_restore-ploteig.patch
]

dependencies = [
    ('GSL', '2.6'),
    ('Perl', '5.32.0'),
]

start_dir = 'src'

# Run "make install" after make to copy all binaries to the bin dir
buildopts = ' && make install'

files_to_copy = ['bin', 'CONVERTF', 'EIGENSTRAT', 'POPGEN', 'README']

fix_perl_shebang_for = ['bin/*.perl', 'bin/ploteig']

sanity_check_paths = {
    'files': ['bin/%s' % x for x in ["baseprog", "convertf", "eigenstrat", "eigenstratQTL", "ploteig"]],
    'dirs': []
}

moduleclass = 'bio'
```

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
@boegelbot please test @ generoso
@boegelbot please test @ generoso

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in total)
node3108.skitty.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/boegel/c5b48dc5c45a35a790fc781c48ac6d15 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @DirectorSloan!

**Pull request merged**